sql/sql_yacc.yy

```diff
@@ -14673,6 +14674,7 @@ ident_keywords_unambiguous:
         | INDEX_STATS_SYM
         | INDEXES
         | INITIAL_SIZE_SYM
+        | INNODB_SYM
```

Comment:
Why do we need to register `INNODB_SYM` as an `ident_keywords_unambiguous` here.
In original MySQL 8.0 syntax
https://dev.mysql.com/doc/refman/8.0/en/alter-instance.html
`ALTER INSTANCE ROTATE INNODB MASTER KEY` was working without this

Reply:
ALTER INSTANCE ROTATE INNODB MASTER KEY is working without this because INNODB is a variable here - it can be either INNODB or BINLOG. Rest of the words in this command are defined as symbols. In our case we only allow ALTER INSTANCE ROTATE INNODB SYSTEM KEY - so there is no reason why not make an INNODB defined as a symbol.

Reply:
@robgolebiowski I am quiet confused here - I do not see `INNODB_SYM` appearing in the new alternatives you added.
Did you want to add
```
          | ROTATE_SYM INNODB_SYM SYSTEM_SYM KEY_SYM ulong_num
          | ROTATE_SYM INNODB_SYM SYSTEM_SYM KEY_SYM
```
instead of
```
          | ROTATE_SYM ident_or_text SYSTEM_SYM KEY_SYM ulong_num
          | ROTATE_SYM ident_or_text SYSTEM_SYM KEY_SYM
```
?
Moreover, are we potentially considering adding support for `ALTER INSTANCE ROTATE BINLOG SYSTEM KEY` construct in future?